pkg/app/config.go

```diff
@@ -46,6 +47,13 @@ func CreateConfig(configFile string) (Config, error) {
 		return Config{}, err
 	}
 
+	if config.Openshift.Namespace == "" {
+		if dat, err = ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace"); err != nil {
+			return Config{}, err
+		}
+		config.Openshift.Namespace = string(dat)
+	}
```

Comment:
Don't necessarily, that's just where I knew I could find the broker namespace. Is there a better place I should look?